cylc/uiserver/handlers.py

```diff
@@ -386,6 +390,18 @@ def open(self, *args, **kwargs):  # noqa: A003
         )
 
     async def on_message(self, message):
+        with suppress(ValueError, SyntaxError):
+            message_dict = ast.literal_eval(message)
+            op_name = ''
+            op_id = message_dict.get("id", None)
+            with suppress(KeyError):
+                op_name = message_dict['payload']['operationName']
+            if (message_dict['type'] == 'start'
+                and op_name
+                    and message_dict['payload']['operationName'] == 'LogData'):
+                self.sub_statuses[op_id] = 'start'
+            if message_dict['type'] == 'stop' and op_id in self.sub_statuses:
+                self.sub_statuses[op_id] = 'stop'
```

Comment:
FYI I think this work should probably go into [graphql-ws](https://github.com/graphql-python/graphql-ws) itself, we should try to do that one day.